composer/utils/analytics_helpers.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Helpers for logging analytics with the MosaicMLLogger."""
```

Comment:
this file should be in the test mosaicml logger file -- the tests ~try to mirror code structure

Reply:
[is this good](https://github.com/mosaicml/composer/blob/angel/add-data-to-metadata-for-analytics/tests/loggers/test_mosaicml_logger.py#L82) @mvpatel2000 ? or did you mean something else?